routes/api.js

```diff
@@ -1,10 +1,28 @@
 const express = require('express');
 const router = express.Router();
 
+const models = require('../database/models');
+
+const localUser = models.user_account;
 /* GET api listing. */
 router.get('/getUserInfo', (req, res) => {
     res.status(200).json({user: req.user});
 });
 
-module.exports = router;
+router.get('/username', (req, res) => {
+    const email=req.user.email;
+    localUser.findAll({
+        where:{
+            email:email
+        }}).then(data => {
+            const username = data[0].fname+ ' '+ data[0].lname;
```

Comment:
Yes I did run, it was giving an error when I switched to the template format. Will try again.